### DISCUSSION TITLE
【我推的孩子】无法添加订阅

### DISCUSSION THREAD

**Discussion by jinhua523** [jxxghp/MoviePilot#2602] [问题咨询]
WARNING】2024-07-24 14:29:42,967 - subscribe.py - 未识别到媒体信息，标题：【我推的孩子】，tmdbid：203737，doubanid：None
【WARNING】2024-07-24 14:29:42,967 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 14:29:42,963 - chain - 请求模块执行：recognize_media ...
【INFO】2024-07-24 14:29:42,962 - subscribe.py - 开始添加订阅，标题：【我推的孩子】 ...
【DEBUG】2024-07-24 12:00:30,403 - chain - 请求模块执行：webhook_parser ...
【INFO】2024-07-24 12:00:00,419 - command.py - 处理事件：notice.message - []
【DEBUG】2024-07-24 11:59:52,773 - chain - 请求模块执行：scheduler_job ...
【WARNING】2024-07-24 11:59:38,334 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 11:59:38,333 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 11:59:38,330 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 11:59:37,102 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 11:59:37,102 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 11:59:37,098 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 11:55:02,686 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 11:55:02,685 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 11:55:02,682 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 11:55:01,116 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 11:55:01,115 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 11:55:01,111 - chain - 请求模块执行：recognize_media ...
【INFO】2024-07-24 11:53:08,195 - plugin.py - 共获取到 156 个线上插件
【DEBUG】2024-07-24 11:51:40,383 - download.py - 媒体库中已存在部分剧集，缺失：{203737: {1: NotExistMediaInfo(season=1, episodes=[10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24], total_episode=24, start_episode=10)}}
【INFO】2024-07-24 11:51:40,381 - emby - 【我推的孩子】 (2023) 媒体库中已存在：{1: [1, 2, 3, 4, 5, 6, 7, 8, 9]}
【DEBUG】2024-07-24 11:51:40,375 - chain - 请求模块执行：media_exists ...
【DEBUG】2024-07-24 11:51:40,373 - tmdbapi.py - 正在获取相似电视剧：203737...
【DEBUG】2024-07-24 11:51:40,371 - tmdbapi.py - 正在获取推荐电视剧：203737...
【DEBUG】2024-07-24 11:51:40,369 - tmdbapi.py - 正在获取电视剧演职人员：203737...
【DEBUG】2024-07-24 11:51:40,367 - chain - 请求模块执行：tmdb_tv_similar ...
【DEBUG】2024-07-24 11:51:40,366 - chain - 请求模块执行：tmdb_tv_recommend ...
【DEBUG】2024-07-24 11:51:40,364 - chain - 请求模块执行：tmdb_tv_credits ...
【DEBUG】2024-07-24 11:51:40,308 - chain - 请求模块执行：obtain_images ...
【DEBUG】2024-07-24 11:51:40,307 - tmdbapi.py - 正在获取剧集组年份：6685b6f6feec89abe3026faa...
【INFO】2024-07-24 11:51:40,307 - themoviedb - 203737 TMDB识别结果：电视剧 【我推的孩子】 (2023)
【DEBUG】2024-07-24 11:51:40,306 - tmdbapi.py - 203737 查询结果：【我推的孩子】
【DEBUG】2024-07-24 11:51:40,305 - tmdbapi.py - 正在查询TMDB电视剧：203737 ...
【DEBUG】2024-07-24 11:51:40,301 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 11:51:37,728 - subscribe.py - 未识别到媒体信息，标题：【我推的孩子】，tmdbid：203737，doubanid：None
【WARNING】2024-07-24 11:51:37,727 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 11:51:37,724 - chain - 请求模块执行：recognize_media ...
【INFO】2024-07-24 11:51:37,723 - subscribe.py - 开始添加订阅，标题：【我推的孩子】 ...
【DEBUG】2024-07-24 11:51:37,708 - download.py - 媒体库中已存在部分剧集，缺失：{203737: {1: NotExistMediaInfo(season=1, episodes=[10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24], total_episode=24, start_episode=10)}}
【INFO】2024-07-24 11:51:37,706 - emby - 【我推的孩子】 (2023) 媒体库中已存在：{1: [1, 2, 3, 4, 5, 6, 7, 8, 9]}
【DEBUG】2024-07-24 11:51:37,698 - chain - 请求模块执行：media_exists ...
【DEBUG】2024-07-24 11:51:37,590 - tmdbapi.py - 正在获取剧集组年份：6685b6f6feec89abe3026faa...
【INFO】2024-07-24 11:51:37,487 - themoviedb - 203737 TMDB识别结果：电视剧 【我推的孩子】 (2023)
【DEBUG】2024-07-24 11:51:37,486 - tmdbapi.py - 203737 查询结果：【我推的孩子】
【DEBUG】2024-07-24 11:51:37,485 - tmdbapi.py - 正在查询TMDB电视剧：203737 ...
【DEBUG】2024-07-24 11:51:37,481 - chain - 请求模块执行：recognize_media ...
【DEBUG】2024-07-24 11:51:37,462 - tmdbapi.py - 203737 查询结果：【我推的孩子】
【DEBUG】2024-07-24 11:51:36,850 - tmdbapi.py - 正在查询TMDB电视剧：203737 ...
【DEBUG】2024-07-24 11:51:36,849 - chain - 请求模块执行：tmdb_seasons ...
【INFO】2024-07-24 11:51:35,226 - media.py - 我推的孩子 搜索到 2 条相关媒体信息
【DEBUG】2024-07-24 11:51:35,223 - chain - 请求模块执行：search_medias ...
【INFO】2024-07-24 11:51:35,221 - media.py - 开始搜索媒体信息：我推的孩子
【INFO】2024-07-24 11:51:35,186 - media.py - 我推的孩子 搜索到 2 条相关媒体信息
【DEBUG】2024-07-24 11:51:34,749 - chain - 请求模块执行：search_medias ...
【INFO】2024-07-24 11:51:34,748 - media.py - 开始搜索媒体信息：我推的孩子
【INFO】2024-07-24 11:51:21,034 - command.py - 用户 WuJinHua 清理缓存 执行完成

**Comment by jinhua523** [comment]
【WARNING】2024-07-24 14:36:51,779 - media.py - 【我推的孩子】 - S01E01 - 第 1 集 - ADWeb.mkv 未识别到媒体信息
【WARNING】2024-07-24 14:36:51,779 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 14:36:51,775 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 14:36:46,500 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 14:36:46,497 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 14:36:37,790 - media.py - /volume1/NASTOOL/CD2/link115/115/Sylink/剧综漫/动漫/【我推的孩子】 (2023)/Season 1/【我推的孩子】 - S01E01 - 第 1 集 - ADWeb.mkv 未识别到媒体信息
【WARNING】2024-07-24 14:36:37,790 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 14:36:37,786 - chain - 请求模块执行：recognize_media ...
【INFO】2024-07-24 14:36:37,784 - media.py - 开始识别媒体信息，文件：/volume1/NASTOOL/CD2/link115/115/Sylink/剧综漫/动漫/【我推的孩子】 (2023)/Season 1/【我推的孩子】 - S01E01 - 第 1 集 - ADWeb.mkv ...
【WARNING】2024-07-24 14:36:09,451 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 14:36:09,451 - themoviedb - 识别媒体信息时未提供元数据名称
【DEBUG】2024-07-24 14:36:09,447 - chain - 请求模块执行：recognize_media ...
【WARNING】2024-07-24 14:36:08,159 - media.py - 【我推的孩子】 未识别到媒体信息
【WARNING】2024-07-24 14:36:08,158 - themoviedb - 识别媒体信息时未提供元数据名称



感觉是因为【】这个括号的问题

**Comment by cng10001** [comment]
是的群里看到有人反映过，改下标题名试试 https://www.themoviedb.org/tv/203737/translations

**Comment by markddd268** [comment]
tmdb数据被改了，改来改去的
